utils/include/core/PropertyType.h

```diff
@@ -365,8 +49,6 @@ enum class PropertyTypeCode : int64_t {
   PORT = 6
 };
```

Comment:
I think the `PropertyTypeCode` class can be deleted, as well.

Reply:
👍 https://github.com/apache/nifi-minifi-cpp/pull/1926/commits/5cc90f1c11217bb00d610ee284e3dfa94da696c5